### ISSUE TITLE
[A11y] Homepage | About Us | Local Contexts Labels and Notices

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#3262]
### Operating System

MacOS

### Browser

Safari

### Affected page

Homepage

### Admin user?

No

### Issue Description

Link: https://oregondigital.org/local-contexts?locale=en

I ran the page on WAVE and recieved 32 errors. The issues revolve around missing alt tags.

### Suggested Fix

Please validate the page on WAVE: https://wave.webaim.org/

See the 32 errors listed and suggested fix on the results page. The images need alt tags.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Run the page again on WAVE and check that the screen reader is reading the image alt tags.
